### ISSUE TITLE
Modifying roads once there are decent sized buildings and activities around it crashes the game (maybe cars?)

### ISSUE THREAD

**Issue by reenenlaurie** [citybound/citybound#311]
[cb_last_error.txt](https://github.com/citybound/citybound/files/2420346/cb_last_error.txt)

I connected an existing road through a longer road with semi established farms and buildings around them.

**Comment by aeickhoff** [comment]
Yes, this is very likely related to cars going on roads being modified. I updated the title. Thanks for the report!
